### DISCUSSION TITLE
Feedack "Design improvements to GitHub Actions navigation"

### DISCUSSION THREAD

**Discussion by mfn** [community/community#36104] [Actions]
Hi,
referring to https://github.blog/changelog/2022-10-13-design-improvements-to-github-actions-navigation/ , here some feedback of my negative experience:

- I've multiple repos where I've more workflows than now shown by default, so I've "show more workflows…" constantly. This means to locate a given workflow by name, I need at least 2 steps now, because even "in browser" page search won't show results in the collapsed area (obviously)
- once navigating to a workflow, it's *always* shown at the top
  This is very confusing. I might have shown related workflows next to each other and might want to navigate between them. But a) the chosen workflow is always shown at the top and not in the list where it was shown before and b) the area is shown collapsed again

TL;DR: I wish a) the collapse/expand would not be there at all and b) don't "move" around the current workflow, this is very irritating.

thx

**Comment by lukasz-mitka** [comment]
Same as above.
Otherwise I like the changes.

**Comment by cmyllis** [comment]
Same here. It still unthinkable how you cannot organize your actions in any way (categories, order etc).
Example of good UI (from TeamCity)
![image](https://user-images.githubusercontent.com/96247992/195799901-dfc45d4c-0d8e-49b7-ac7b-1b96817bdfa6.png)
- Grouped by categories
- Expandable/collapsable
- Option to configure what you want to see (per user options)

**Comment by freddydk** [comment]
Same here, with our set of templates called AL-Go for GitHub, we have a series of workflows, which is larger than the new optiomization:
![image](https://user-images.githubusercontent.com/10775043/198864709-6a47a251-f89c-4b37-96c7-006701d44f20.png)

Would be nice to be able to group the workflows instead of just having one "all workflows" group.

**Comment by rcdailey** [comment]
Why am I not seeing the new design in my repository?
